src/components/Pages/Discovery/Components/MasonryGrid.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="grid" ref="grid">
+    <slot/>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'MasonryGrid',
+  data () {
+    return {
+      timer: null
+    }
+  },
+  methods: {
+    resizeCards () {
```

Comment:
This is totally fine, no need to change but I was wondering if there was a more native vuetify way to do this (e.g. maybe using it's breakpoint functionality)